src/olympia/shelves/serializers.py

```diff
@@ -34,13 +34,14 @@ class Meta:
             'title',
             'url',
             'endpoint',
+            'addon_type',
```

Comment:
specify the field above like `AddonSerializer` does https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/serializers.py#L444

Reply:
(that will do the conversion from int to a string value)